assets/web/apps.html

```diff
@@ -1,95 +1,110 @@
+              <button class="btn btn-sm w-100 btn-outline-danger" title="Delete this application" @click="showDeleteForm(i)">Delete</button>
+            </div>
           </td>
         </tr>
       </tbody>
     </table>
+    <p>
+      <button type="button" class="btn btn-primary" @click="newApp" v-bind:disabled="isLoading">Create new application</button>
+    </p>
   </div>
-  <div class="edit-form card mt-2" v-if="showEditForm">
-    <div class="p-4">
-      <!--name-->
-      <div class="mb-3">
-        <label for="appName" class="form-label">Application Name</label>
-        <input type="text" class="form-control" id="appName" aria-describedby="appNameHelp" v-model="editForm.name">
-        <div id="appNameHelp" class="form-text">Application Name, as shown on Moonlight</div>
-      </div>
-      <!--output-->
-      <div class="mb-3">
-        <label for="appOutput" class="form-label">Output</label>
-        <input type="text" class="form-control  monospace" id="appOutput" aria-describedby="appOutputHelp"
-          v-model="editForm.output">
-        <div id="appOutputHelp" class="form-text">The file where the output of the command is stored, if it is not
-          specified, the output is ignored</div>
-      </div>
-      <!--prep-cmd-->
-      <div class="mb-3 d-flex flex-column">
-        <label for="appName" class="form-label">Command Preparations</label>
-        <div class="form-text">A list of commands to be run before/after the application. <br> If any of the
-          prep-commands fail, starting the application is aborted</div>
-        <table v-if="editForm['prep-cmd'].length > 0">
-          <thead>
-            <th class="precmd-head">Do</th>
-            <th class="precmd-head">Undo</th>
-            <th style="width: 48px;"></th>
-          </thead>
-          <tbody>
-            <tr v-for="(c,i) in editForm['prep-cmd']">
-              <td><input type="text" class="form-control monospace" v-model="c.do"></td>
-              <td><input type="text" class="form-control monospace" v-model="c.undo"></td>
-              <td><button class="btn btn-danger" @click="editForm['prep-cmd'].splice(i,1)">&times;</button></td>
-            </tr>
-          </tbody>
-        </table>
-        <button class="mt-2 btn btn-success" style="margin: 0 auto;" @click="addPrepCmd">&plus; Add</button>
-      </div>
-      <!--detatched-->
-      <div class="mb-3">
-        <label for="appName" class="form-label">Detached Commands</label>
-        <div v-for="(c,i) in editForm.detached" class="d-flex justify-content-between my-2">
-          <pre>{{c}}</pre>
-          <button class="btn btn-danger mx-2" @click="editForm.detached.splice(i,1)">&times;</button>
-        </div>
-        <div class="d-flex justify-content-between">
-          <input type="text" class="form-control monospace" v-model="detachedCmd">
-          <button class="btn btn-success mx-2" @click="editForm.detached.push(detachedCmd);detachedCmd = '';">+</button>
```

Comment:
Adding a :disabled="detatchedCmd.length == 0" in the button would avoid adding empty detatched commands